broker/src/test/java/io/zeebe/broker/system/configuration/BrokerCfgTest.java

```diff
@@ -240,16 +240,20 @@ public void shouldEnableDebugLogExporter() {
     environment.put(ENV_DEBUG_EXPORTER, "true");
 
     // then
-    assertDefaultDebugLogExporter(false);
+    assertDefaultDebugLogExporter();
   }
 
   @Test
-  public void shouldEnableDebugLogExporterWithPrettyOption() {
+  public void shouldDisableDebugLogExporter() {
```

Comment:
Is there a test case that verifies the debug exporter is not enabled if the environment variable doesn't exist?

Reply:
No, and honestly I was on the fence about the worth of this test - on one hand, the normal state is to not have the exporter. On the other hand, previous versions would include it regardless of the value, so I did want a "regression" test to ensure the behaviour was changed.

However, now that I think about it, we could simply have a test which checks there are no exporters by default (we have several other default tests, but none for exporters).

Reply:
Added the test and a follow up issue: https://github.com/zeebe-io/zeebe/issues/5944